### PR TITLE
Copyright bump to 2021

### PR DESCRIPTION
**Pull request by obilodeau**
Some files were modified further. I went through all changed files with `git diff HEAD HEAD@{2020-12-31} --name-only` and verified individual history with `git log -p <filename>`.

### CODE REVIEW

File: pyrdp/layer/layer.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #
 # This file is part of the PyRDP project.
-# Copyright (C) 2018 GoSecure Inc.
+# Copyright (C) 2018, 2019, 2021 GoSecure Inc.
```
Comment (xshill) [inline pyrdp/layer/layer.py:3]: Can't we just write 2018-2021? Here and a couple other places.

Comment (xshill) [reply pyrdp/layer/layer.py:3]: (basically everywhere we have more than 1 year)

**Review by xshill — CHANGES_REQUESTED** [review]

File: pyrdp/layer/layer.py (hunk 1 above)
Comment (obilodeau) [reply pyrdp/layer/layer.py:3]: Unfortunately no, where I wrote without dashes are places where there were no code changes in the missing years. I'm not sure if we can apply the copyright statement if there was no change.

Comment (xshill) [reply pyrdp/layer/layer.py:3]: ah ok then

**Pull request merged**